app/Providers/AppServiceProvider.php

```diff
@@ -21,6 +21,9 @@ public function register(): void
             $clientId = substr(str_shuffle(MD5(microtime())), 0, $maxClientIdLength);
             $client = new Client(env('MQTT_SERVER'), env('MQTT_PORT'), $clientId);
             $client->setAuthDetails(env('MQTT_USER'), env('MQTT_PASSWORD'));
+            if (env('MQTT_TLS', false)) {
```

Comment:
Please add a new line before the if-statement.  Also, to be consistent with the rest of the codebase, could you please write the if-statement like:

```
$tlsEnabled = env('MQTT_TLS', false);

if ($tlsEnabled) {
    ...
}
```

This if-statement is not covered by any tests.  It will be necessary that it is covered by adding test(s) with the environment variable set to both true and false before merging this PR.

https://coveralls.io/builds/13544578/source?filename=app%2FProviders%2FAppServiceProvider.php